app/default/content_routes.py

```diff
@@ -48,19 +48,43 @@ def cof_r2w2_all_questions_redirect():
 
 @content_bp.route("/contact_us", methods=["GET"])
 def contact_us():
-    current_app.logger.info("Contact us page loaded.")
     fund_short_name = request.args.get("fund")
     round_short_name = request.args.get("round")
-    if not (fund_short_name and round_short_name):
-        (fund_short_name, round_short_name) = get_default_fund_and_round()
-    round_data = get_round_data_fail_gracefully(
-        fund_short_name, round_short_name, True
+    current_app.logger.info(
+        f"Contact us page loaded for fund {fund_short_name} round"
+        f" {round_short_name}."
     )
-    fund_data = get_fund_data_by_short_name(fund_short_name)
+    if round_short_name and fund_short_name:
+        round_data = get_round_data_fail_gracefully(
```

Comment:
fail_gracefully, not seen this before. Is it just error handling?

Reply:
Yeah that method has been there for a while - it just returns an empty Round object if it fails, rather than an error, so the contact us page knows to display default contact info.